### DISCUSSION TITLE
Example Needed

### DISCUSSION THREAD

**Discussion by gdelfino** [DerYeger/yeger#283] [General]
The website provides a good example to decide if this library is a good fit for one needs. However, there is no example to actually get started using d3-graph-controller. It would be great if there was an option to download that same "see it in action" example in order to get started which I'm finding difficult.

**Comment by DerYeger** [comment]
You can find the source code of the demo [here](https://github.com/DerYeger/yeger/blob/main/docs/d3-graph-controller-docs/.vitepress/components/Demo.vue) or check out the [guide](https://graph-controller.yeger.eu/guide/).
